infra/main.bicep

```diff
@@ -177,6 +177,8 @@ param authType string = 'keys'
 ])
 param hostingModel string = 'container'
 
+@description('Use Integrated Vectorization')
```

Comment:
Can we keep this with the other AI Search params?

Reply:
Updated